src/applications/vaos/reducers/appointments.js

```diff
@@ -10,11 +10,13 @@ import {
   FETCH_PAST_APPOINTMENTS_FAILED,
 } from '../actions/appointments';
 
+import { FETCH_STATUS } from '../utils/constants';
+
 const initialState = {
   confirmed: null,
   confirmedLoading: true,
   pending: null,
-  pendingLoading: true,
+  pendingStatus: FETCH_STATUS.notStarted,
```

Comment:
@narin When I started implementing the loading logic it started to feel like a loading boolean wasn't going to be quite enough to handle everything. So I made it a status with a few options, so we can have more specific control over what to show in the UI for different states.

If this makes sense to you, I'll update confirmed in this PR as well. I think we can leave your current selector page PR as is, so we can get that overall Redux logic in place more quickly.

Reply:
Sure, this makes sense.  The more accurate status of the fetch we can store the better.